OpenLaTeXStudio/EditorModule/src/latexstudio/editor/EditorTopComponent.java

```diff
@@ -82,21 +83,26 @@ public void propertyChange(PropertyChangeEvent evt) {
                     autoCompletion.setAutoActivationDelay((Integer) evt.getNewValue());
                 } else if (evt.getPropertyName().equals(ApplicationSettings.LINEWRAP_STATUS) && rSyntaxTextArea != null) {
```

Comment:
To be honest, I'd feel more confident merging those changes, once all of the settings are migrated to the new system. At this point it's not quite easy for me to figure out how the final result will look like. I can even hold off all other tickets adding/editing properties for the sake of finishing this first (we do not have many of them anyways).